definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx

```diff
@@ -39,14 +39,13 @@ config {
         latest_cpd_lead_provider_name: "Name of Lead Provider associated with participant's latest declaration.",
         latest_declaration_date: "Date evidence was submitted for participant's latest declaration.",
         latest_declaration_updated_at: "Date participant's latest declaration was updated",
-        participant_profile_id: "The participant profile ID provided with a participant's declaration(s).",
         application_status: "Indicates whether the application has been approved by the lead provider.Possible fields: Pending, Accepted, Rejected.",
         updated_at: "Refers to the latest updated date for the application record.",
         employer_name: "Shows the entered employer name of non-school users. Text field.",
         employment_role: "Shows the employment role of non-school users. Text field.",
         employment_type: "For non-school users, shows the type of institution they are employed in. Possible fields: Local authority supply teachers, Local authority virtual school, Young offender institute, Hospital school, Other.",
         declaration_user_id: "This is the user ID provided with a participant's declaration(s). User ID also exists independently in other tables.",
-        profile_user_id: "The ID from a participant's profile. Participant profiles are only generated when an application is accepted by a lead provider.",
+        participant_profile_id: "The ID from a participant's profile in the CPD service. Participant profiles are only generated when an application is accepted by a lead provider. There is one profile per accepted application. An NPQ participant will have a different participant_profile_id for each accepted NPQ. Their participant_profile_id should match the application_id and should match the participant_profile_id available in the declarations_latest table. While their participant_profile_id changes across NPQs, their participant_identity_id will remain the same across their participant profiles.",

```

Comment:
"their participant_identity_id will remain the same across their participant profiles **so long as they have signed up with the same email**"